server/spec/filter_response_spec.rb

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 require 'spec_helper'
 require 'candlepin_scenarios'
+require 'candlepin_api.rb'
```

Comment:
`candlepin_scenarios` already requires `candlepin_api` so it's available transitively. no need for this require

Reply:
Done.